examples/with-apollo/README.md

```diff
@@ -1,6 +1,6 @@
 # Apollo Example
 ## Demo
-https://with-apollo-oeaizzfwlu.now.sh
+https://with-apollo-ehxkwxrnvf.now.sh
```

Comment:
would that just be `now alias https://with-apollo-ehxkwxrnvf.now.sh [domain name]` ?